tests/storage/test_storage_implementation.py

```diff
@@ -0,0 +1,68 @@
+"""Tests the SqlStorage methods that are NOT tested through the
+REST API tests.  To test against different SQL backends, this
+test must be run with different ANYVAR_TEST_STORAGE_URI settings
+and different ANYVAR_SQL_STORE_BATCH_ADD_MODE settings
+"""
+
+from anyvar.storage.abc import StoredObjectType
+from anyvar.translate.vrs_python import VrsPythonTranslator
+
+
+# Test add_objects method (replaces __setitem__)
+def test_add_objects(storage, alleles):
```

Comment:
These individual cases appear to be interdependent. Minor nit but my personal preference would be towards independence.